docs/plugins/tars.md

```diff
@@ -11,23 +11,31 @@
 
 为了解决该问题 GitHub 提供了一个名为 `Require branches to be up to date before merging` 的分支保护选项。打开该选项之后，PR 只有在使用最新 Base 分支的时候才能合并。**虽然这样能解决这个问题，但是需要人为手动的去点击 GitHub 按钮合并最新的 Base 分支到 PR，这是个机械且重复的事情**。
 
-ti-community-tars 就是为了解决该问题而设计，它会在 PR 有回复或者更新时自动检测 PR 是否过期，然后帮助我们自动的合并最新的 Base 分支到 PR。除此之外，它也支持定期扫描所有配置了该插件的仓库的所有 PR，然后帮助我们更新已经过期的 PR。
+ti-community-tars 就是为了解决该问题而设计，它会在 PR 回复、更新或者 Base 分支有新提交时帮助我们自动合并最新的 Base 分支到 PR。除此之外，它也支持定期扫描所有配置了该插件的仓库的所有 PR，对所有的 PR 进行更新。
 
 ## 设计思路
 
-实现该插件，我们不仅需要在 PR 更新或者有回复时，去自动检测和更新。还要考虑到某些 PR 在 code review 结束之后不会再进行更新和回复，这些 PR 可能就会因为 `Require branches to be up to date before merging` 的选项要求无法合并。所以该插件还需要支持定期的扫描更新这些 PR。
+实现该插件需要考虑以下几种情形：
+- PR 有回复或者更新
+  - 当 PR 有回复或者更新时说明有人在关注该 PR，可能希望该 PR 尽快的合并，所以我们要尽快的响应并更新该 PR
+- Base 分支有新的提交
+  - 当 Base 分支有了新的提交之后，我们应该尽快查找其他可以合并的 PR 并将最新的 Base 更新到 PR 当中
```

Comment:
How can tars judge whether PR can be merged?

Reply:
can-merge label.